src/Illuminate/Queue/Worker.php

```diff
@@ -122,6 +122,9 @@ public function daemon($connectionName, $queue, WorkerOptions $options)
 
         $lastRestart = $this->getTimestampOfLastQueueRestart();
 
+        $timer = microtime(true);
```

Comment:
You should use the new hrtime function, because microtime could go backwards in time if the system clock changes (which actually can happen a lot: my mac changes its clock multiple time each hour by a few seconds!).